core/solver/idr.cpp

```diff
@@ -170,6 +170,25 @@ void Idr<ValueType>::iterate(const LinOp *b, LinOp *x) const
 
     int total_iter = -1;
 
+    /* Memory movement summary for iteration with subspace dimension s
+     * Per iteration:
+     * (11/2s+10+7/(s+1))n * values + matrix/preconditioner storage
+     * For (s+1) iterations:
+     * (11/2s^2+31/2s+17)n * values + (s+1) * matrix/preconditioner storage
```

Comment:
add some description on this? each solver iteration needs around this memory movement